### PR TITLE
Update

### PR DESCRIPTION
**Pull request by DJPlaya**
*Fixed non-CSGO Games will not work
*Updated Syntax
*Fixed Typos

### CODE REVIEW

File: scripting/togglemusic_dhook.sp [hunk 1]
```diff
@@ -33,81 +27,89 @@ StringMap g_smCommon;
...
 StringMap g_smVolume;
 
-public Plugin myinfo =
+public Plugin myinfo = 
 {
-	name = PLUGIN_NAME,
-	author = "Mitch & Agent Wesker",
-	description = "Allows clients to toggle ambient sounds played by the map",
-	version = PLUGIN_VERSION,
+	name = "Toggle Music",
```
Comment (sneak-it) [inline scripting/togglemusic_dhook.sp:32]: Why was this change needed? You remove the PLUGIN_NAME define but maintain PLUGIN_VERSION.

File: scripting/togglemusic_dhook.sp [hunk 2]
```diff
@@ -1,17 +1,11 @@
...
-
-#include <sourcemod>
 #include <sdktools>
 #include <sdkhooks>
 #include <dhooks>
 #include <clientprefs>
 #include <regex>
 //#include <emitsoundany>
 
-#pragma newdecls required
```
Comment (sneak-it) [inline scripting/togglemusic_dhook.sp:11]: Why would you remove the requirement to use new-style declarations?

File: scripting/togglemusic_dhook.sp [hunk 3]
```diff
@@ -1,17 +1,11 @@
-#pragma semicolon 1
-
-#include <sourcemod>
```
Comment (sneak-it) [inline scripting/togglemusic_dhook.sp:3]: why are you removing the basic sm include?

File: scripting/togglemusic_dhook.sp [hunk 4]
```diff
@@ -33,81 +27,89 @@ StringMap g_smCommon;
...
 
-public Plugin myinfo =
+public Plugin myinfo = 
 {
-	name = PLUGIN_NAME,
-	author = "Mitch & Agent Wesker",
-	description = "Allows clients to toggle ambient sounds played by the map",
-	version = PLUGIN_VERSION,
+	name = "Toggle Music", 
+	author = "Mitch & Agent Wesker (Playa Edit)",
```
Comment (sneak-it) [inline scripting/togglemusic_dhook.sp:33]: It's not exactly good practice to put your name in the credits for this small of a change

File: scripting/togglemusic_dhook.sp [hunk 5]
```diff
@@ -33,81 +27,89 @@ StringMap g_smCommon;
...
-		HookEvent("round_poststart", Event_PostRoundStart);
 		
+		if (temp == null)
+			SetFailState("Gamedata Missing!");
+			
+		if (eVer == Engine_CSGO)
+			HookEvent("round_poststart", Event_RoundStarted); // CSGO Only
+			
+		else
+			HookEvent("round_start", Event_RoundStarted); // Supported on all Games
```
Comment (sneak-it) [inline scripting/togglemusic_dhook.sp:111]: Isn't round_start a bit too early in many instances, hence the need for round_poststart? The intention is good, however this should require testing to actually confirm this change is needed, or even works.

On the other hand, this could also be used and specific games might not need to be defined, as round_start might be good enough for all. Generally speaking, if round_start works for one game, it should work for most/all.

**Review by sneak-it — CHANGES_REQUESTED** [review]
The majority of the changes in this PR are simply subjective formatting changes. The only functionality change is hooking round_start on all games except CS:GO, which would require testing across all specified games.

File: scripting/togglemusic_dhook.sp (hunk 1 above)
Comment (DJPlaya) [reply scripting/togglemusic_dhook.sp:32]: Ime a Freak of Perfectness, the Plugin Name is only used once, the Version 2 Times. This is just optimization work making it look better, there is no actual relevant change.
I even remove all the Spaces between if Arguments and the Brackets in all Plugins i maintain.

File: scripting/togglemusic_dhook.sp (hunk 2 above)
Comment (DJPlaya) [reply scripting/togglemusic_dhook.sp:11]: Which Editor do you use? I have a VS Code Plugin and SPEdit, both mark old styled Code.

File: scripting/togglemusic_dhook.sp (hunk 3 above)
Comment (DJPlaya) [reply scripting/togglemusic_dhook.sp:3]: SM dosent needs to be included in order for the Plugin to work, it compiles fine on my Side.

File: scripting/togglemusic_dhook.sp (hunk 4 above)
Comment (DJPlaya) [reply scripting/togglemusic_dhook.sp:33]: Remove it if you like or give me more Stuff to change :P

File: scripting/togglemusic_dhook.sp (hunk 5 above)
Comment (DJPlaya) [reply scripting/togglemusic_dhook.sp:111]: If its too early we could use RequestFrame2, which is RequestFrame just with the mount of Frames to specify. The Forum post Title starts with "[ANY]" meaning that it works with most HL Mods aswell as all updated Valve Games. CSS and TF2 should be fine for testing, if it makes Problems with other Mods, a Ticket will be opened.
